Kubernetes/omsagent.yaml

```diff
@@ -327,7 +327,7 @@ spec:
       serviceAccountName: omsagent
       containers:
         - name: omsagent
-          image: "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod05222020"
+          image: "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod05262020"
```

Comment:
can you also update the docker provider version to 10.0.0-1 ?

Reply:
> can you also update the docker provider version to 10.0.0-1 ?

Good catch. Fixed :)